dttools/src/set.c

```diff
@@ -174,6 +202,28 @@ int set_insert(struct set *s, const void *element)
 	return 1;
 }
 
+int set_insert_set(struct set *s, struct set *s2)
+{
+	set_first_element(s2);
+	const void *element;
+	while((element = set_next_element(s2))){
+		set_insert(s, element);
+	}
+
+	return 1;
```

Comment:
If we return a number, should we return the number added, or the number of element that now exist in the set. More clearly, what should we return in the element already exists in the set.